Modules/Core/Common/src/itkSmapsFileParser.cxx

```diff
@@ -300,7 +300,7 @@ ITKCommon_EXPORT std::istream &
 template <typename TFirstType>
 struct MapRecordPlusor
 {
-  MapRecordPlusor<TFirstType>(const char * token = "Size")
+  MapRecordPlusor(const char * token = "Size")
```

Comment:
Thanks @darrent1974 Looks fine, of course, with any version of the C++ standard. I do observe a compilation error, indeed, when trying gcc 11.1 `-std=c++20`, at https://godbolt.org/z/14T4E7jGx

```
template <typename T> class Bar
{
public:
  Bar<T>() {}
};

```

It says:
```

source>:8:10: error: expected unqualified-id before ')' token
    8 |   Bar<T>() {}
      |          ^
```

Is that the error you encountered as well? If so, maybe you could mention the error message in the commit message. So that we can easily see (and verify) that it isn't just a cosmetic fix.

BTW I didn't yet know that this is a C++20 language change! Thanks for the information 😃

Reply:
Yes, that's the one I'll change it accordingly.